src/abi/json-abi-format.md

```diff
@@ -35,52 +36,39 @@ The ABI of a contract is represented as a JSON object containing the following p
     - `"name"`: the name of the attribute.
     - `"arguments"`: an array of attribute arguments.
 - `"loggedTypes"`: an array describing all instances of [`log`](../fuel-vm/instruction-set.md#log-log-event) or [`logd`](../fuel-vm/instruction-set.md#logd-log-data-event) in the contract's bytecode. Each instance is a JSON object that contains the following properties:
   - `"logId"`: a string containing the 64bit hash based decimal ID calculated from the first 8 bytes of the `sha256` of a string that represents the type logged as defined in [Hash Based Ids](./hash-based-ids.md). The [`log`](../fuel-vm/instruction-set.md#log-log-event) and [`logd`](../fuel-vm/instruction-set.md#logd-log-data-event) instructions must set their `$rB` register to that ID.
-  - `"loggedType"`: a _type application_ represented as a JSON object that contains the following properties:
-    - `"type"`: the _type declaration_ hash based ID of the type of the value being logged.
-    - `"typeArguments"`: an array of the _type arguments_ used when applying the type of the value being logged, if the type is generic, and `null` otherwise. Each _type argument_ is a _type application_ represented as a JSON object that contains the following properties:
-      - `"type"`: the _type declaration_ hash based ID of the type of the _type argument_.
-      - `"typeArguments"`: an array of the _type arguments_ used when applying the type of the _type argument_, if the type is generic, and `null` otherwise. The format of the elements of this array recursively follows the rules described in this section.
+  - `"concreteTypeId"`: the _type concrete declaration_ hash based ID of the value being logged.
 - `"messagesTypes"`: an array describing all instances of [`smo`](../fuel-vm/instruction-set.md#smo-send-message-to-output) in the contract's bytecode. Each instance is a JSON object that contains the following properties:
-  - `"messageDataType"`: a _type application_ represented as a JSON object that contains the following properties:
-    - `"type"`: the _type declaration_ hash based ID of the type of the message data being sent.
-    - `"typeArguments"`: an array of the _type arguments_ used when applying the type of the message data being sent, if the type is generic, and `null` otherwise. Each _type argument_ is a _type application_ represented as a JSON object that contains the following properties:
-      - `"type"`: the _type declaration_ hash based ID of the type of the _type argument_.
-      - `"typeArguments"`: an array of the _type arguments_ used when applying the type of the _type argument_, if the type is generic, and `null` otherwise. The format of the elements of this array recursively follows the rules described in this section.
+  - `"message_id"`: a unique string ID.
```

Comment:
My apologies for the post-merge feedback, but was there any particular reason why this is snake case? @esdrubal